pymc/model/core.py

```diff
@@ -1740,7 +1738,7 @@ def debug_parameters(rv):
             done = {}
             used_ids = {}
             for i, out in enumerate(rv_inputs.maker.fgraph.outputs):
-                print_(f"{i}: ", end=""),
+                (print_(f"{i}: ", end=""),)
```

Comment:
I think this was a bug

```suggestion
                print_(f"{i}: ", end="")
```